srcpkgs/composer/template

```diff
@@ -10,7 +10,7 @@ license="MIT"
 homepage="https://getcomposer.org/"
 distfiles="https://github.com/composer/composer/releases/download/${version}/composer.phar
  https://raw.githubusercontent.com/composer/composer/master/LICENSE"
```

Comment:
Use `$version` instead of `master` here, that way we avoid breaking the build unnecessarily if something changes in that file.

Reply:
there ya go bud